dashboard/clone_or_pull_all_repos.sh

```diff
@@ -8,38 +8,27 @@ REPO_URL_PREFIX="https://github.com/fabric8-analytics/"
 REPOSITORIES="
 fabric8-analytics-common
 fabric8-analytics-data-model
-fabric8-analytics-jobs
```

Comment:
Do not remove

Reply:
Remove tagger, nvd-toolkit, notification scheduler, release-monitor, cvejob

Reply:
do not remove what 
fabric8-analytics-jobs ?

Reply:
done added it back PTAL